src/18C790.c

```diff
@@ -267,7 +267,7 @@ void btl_state_update_celebration(void) {
                 btl_cam_use_preset(BTL_CAM_PRESET_C);
                 btl_cam_move(5);
                 dma_copy(_79EF40_ROM_START, _79EF40_ROM_END, _79EF40_VRAM);
-                sfx_play_sound(0x80000008);
+                sfx_play_sound(SOUND_80000008);
```

Comment:
I think 0x80000000 is a flag and there aren't unique sounds starting with that value... but I could be wrong. do you know anything about that?

Reply:
0x80000000 is for looping sounds I believe

Reply:
Yes, there are also 0xB00000XX and I think one other. 0x80000008 is distinct from 0x8; it is NOT sound 8 looped. Giving them separate names is correct.